OpenFAST/IEA-15-240-RWT/IEA-15-240-RWT_InflowFile.dat

```diff
@@ -1,52 +1,52 @@
-"Wind/90m_12mps_twr"   FilenameRoot - Rootname of the full-field wind file to use (.wnd, .sum)
+"none"                 FilenameRoot - Rootname of the full-field wind file to use (.wnd, .sum)
 False                  TowerFile   - Have tower file (.twr) (flag)
 ================== Parameters for HAWC-format binary files  [Only used with WindType = 5] =====================
-"wasp/Output/basic_5u.bin" FileName_u  - name of the file containing the u-component fluctuating wind (.bin)
-"wasp/Output/basic_5v.bin" FileName_v  - name of the file containing the v-component fluctuating wind (.bin)
-"wasp/Output/basic_5w.bin" FileName_w  - name of the file containing the w-component fluctuating wind (.bin)
-64                     nx          - number of grids in the x direction (in the 3 files above) (-)
-32                     ny          - number of grids in the y direction (in the 3 files above) (-)
-32                     nz          - number of grids in the z direction (in the 3 files above) (-)
-16.0                   dx          - distance (in meters) between points in the x direction    (m)
-3.0                    dy          - distance (in meters) between points in the y direction    (m)
-3.0                    dz          - distance (in meters) between points in the z direction    (m)
-150.0                  RefHt_Hawc  - reference height; the height (in meters) of the vertical center of the grid (m)
+"none"                 FileName_u  - name of the file containing the u-component fluctuating wind (.bin)
+"none"                 FileName_v  - name of the file containing the v-component fluctuating wind (.bin)
+"none"                 FileName_w  - name of the file containing the w-component fluctuating wind (.bin)
+2                      nx          - number of grids in the x direction (in the 3 files above) (-)
+2                      ny          - number of grids in the y direction (in the 3 files above) (-)
+2                      nz          - number of grids in the z direction (in the 3 files above) (-)
+10                     dx          - distance (in meters) between points in the x direction    (m)
+10                     dy          - distance (in meters) between points in the y direction    (m)
+10                     dz          - distance (in meters) between points in the z direction    (m)
+0.0                    RefHt_Hawc  - reference height; the height (in meters) of the vertical center of the grid (m)
 -------------   Scaling parameters for turbulence   ---------------------------------------------------------
-2                      ScaleMethod - Turbulence scaling method   [0 = none, 1 = direct scaling, 2 = calculate scaling factor based on a desired standard deviation]
+0                      ScaleMethod - Turbulence scaling method   [0 = none, 1 = direct scaling, 2 = calculate scaling factor based on a desired standard deviation]
```

Comment:
Is there a good reason for this change?

Reply:
no, but the value 2 is also not used by anyone. still I reverted to 2